source/exploring_catalog_and_datasets/01_looking_up_the_catalog/catalog.rst

```diff
@@ -1,79 +1,179 @@
+- *Least downloads first*: the datasets are sorted from the least downloaded one to the most downloaded.
+- *Most popular first* (one of the 3 most popular methods "Popular"): the datasets are sorted from the most popular one to the least popular.
+- *Least popular first*: the datasets are sorted from the least popular one to the least popular.
+
+To modify the sorting method, 2 possibilities:
+
+A. Click one of the 3 most popular methods: "Modified", "Popular", "A-Z".
+B. 1. Click the |icon-seemore| button to display all available sorting methods.
+   2. Click the chosen sorting method in the list.
+
+Once the new sorting method is chosen, the catalog automatically updates.
 
 .. _filtering-catalog:
 
 Filtering the catalog
 ---------------------
 
-On the left hand side of the catalog sits the filtering menu. It allows the user to sort the dataset list by date or title, and to filter it with precision.
+When navigating a catalog with only a few datasets, it is easy to scroll down and check all the datasets available. But when navigating catalogs with dozens, even hundreds of datasets, scrolling down isn't an option.
+As an end-user on an open data portal, the reason for visiting that portal may not be just curiosity. The reason for being on the portal may be for a precise purpose ; to find specific data or to search for data from a specific topic, time period or even geographic area.
+
+For these reasons and more, a variety of filtering options are available, below the number of datasets and the catalog sorting option, on the left side of the interface.
+
+.. image:: images/filtering_catalog.png
+   :alt: All options to filter the catalog: search bar, mini map and metadata filters
+
+It is possible to filter:
+
+- by textual research, using the search bar
+- by geographic area, using the mini map
+- by metadata, using the available metadata-based filters
+
+Using the search bar
+^^^^^^^^^^^^^^^^^^^^
+
+The search bar should be used to make textual searches in the catalog. When doing a textual search, the platform will look for a match between the search and the metadata from all the available datasets (title, description, keywords etc.). The catalog will be filtered to only display those matching datasets.
+
+.. admonition:: Note
+   :class: note
+
+   If there is no match after the textual search, the information "Your search didn't match any dataset." will be displayed instead of the catalog cards.
+   Each textual search being recorded, the administrators of an OpenDataSoft portal know about each search without a match. That way, knowing the interests and needs of their end-users, they might publish new datasets in the future to answer these interests and needs.
+
+To make a textual search:
+
+1. Click on the search bar area.
+2. Type the word to search in the datasets.
+3. Press enter for the search to be taken into account, and the catalog filtered according to the typed word.
+
```

Comment:
Could be interesting to add an example of the use of the search bar. For instead, wearch for "world heritage" and see that result differ from the initial screenshot of this chapter.

It would also be interesting to tell users that they can use advanced search filters. You can give a simple example based on the modification date and link to the query language chapter for those who want to know more about this topic.

Reply:
I don’t think I will add an example with a screenshot because there aren’t any other examples in this documentation, and also it would make it longer and “heavier”, if that makes sense. So if that’s alright with you I think I will keep it that way.

However you are right about the link to the query language chapter of the documentation, I am definitely adding this! And actually, maybe the examples you are talking about could be quite relevant in the query language section.

Reply:
Perfect!